raiden/tests/unit/test_raiden_event_handler.py

```diff
@@ -36,39 +39,40 @@ def test_handle_contract_send_channelunlock_already_unlocked():
         token_network_address=token_network_address,
         partner_address=participant,
     )
+    assert channel_state
 
     channel_state.our_state.onchain_locksroot = LOCKSROOT_OF_NO_LOCKS
     channel_state.partner_state.onchain_locksroot = LOCKSROOT_OF_NO_LOCKS
 
     def detail_participants(  # pylint: disable=unused-argument
```

Comment:
You can probably remove the pylint ignore, now that you've prefixed the parameters with and underscore.

Reply:
Nice catch.